Cargo.toml

```diff
@@ -0,0 +1,4 @@
+[workspace]
+members = [
+    "vss-accessor",
```

Comment:
there will be an additional vss-lib added in future (mainly phase-2) for abstracting out additional versioning handling. 
we want to give flexibility to clients to just depend on the basic thin client instead of full lib.

Reply:
I'd prefer a single crate. When you start breaking across crates you lose the ability to use `pub(crate)`/`pub(super)` and make it difficult to implement traits on external types.

Reply:
the fat lib of vss could have significantly more dependencies. 
We might need to depend on some in-memory/file-based database (sled/sqlite) to implement SecondaryKVStore.
That will help with maintaining global-version and sync across multiple-devices. 
For single device-use it is very simple to just directly use vss-accessor (thin-client). Hence the separation. 

regarding pub(crate), there is separation of concern between vss-thin-client and fat-lib, we shouldn't be needing it ideally.